src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/FormStore.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import {action, autorun, observable} from 'mobx';
+import ResourceRequester from '../../../services/ResourceRequester';
+import type {Schema} from '../types';
+
+export default class FormStore {
+    resourceKey: string;
+    id: string;
+    locale = observable();
+    disposer: () => void;
+    @observable loading: boolean = false;
+    @observable saving: boolean = false;
+    @observable data: Object = {};
+    @observable dirty: boolean = false;
+
+    constructor(resourceKey: string, id: string) {
+        this.resourceKey = resourceKey;
+        this.id = id;
+        this.disposer = autorun(this.load);
+    }
+
+    load = () => {
+        if (!this.locale.get()) {
```

Comment:
This doesn't work for forms in which we have no localizations... Therefore we should try to implement another form without localizations (maybe contacts?) quite fast IMO.

Reply:
What about `ghost` content?

Reply:
That's really not handled yet... But I guess that's something that should be implemented in this component. I guess there should be some kind of flag on the `Renderer`, which is passed further down, and decides whether the initial data should be filled as placeholder or real values. But that's just a vague idea, and I would argue it is out of scope of this PR.